docs/UserGuide.adoc

```diff
@@ -158,68 +175,72 @@ You can find a contact added to the address book easily with a name.
 
 Format: `find n\[name]`
 
-[NOTE]
-`[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
-
-Example: +
-`find n\john DOE`
-
-==== Add Picture to Contact: `addpic`
-You can add a profile picture to a contact in your address book.
-
-Format: `addpic n\[name] pa\[path]`
-
 [NOTE]
 ====
 * `[name]` is not case-sensitive. (E.g. "Jane doe" is the same as "jane Doe")
-* Absolute file path is used for `[path]`.
+* You can find partial name
 ====
 
-Example: +
-`addpic n\john doe pa\C:\Users\aloha\Desktop\aloha.png`
+Example:
 
-Sets the profile picture of John Doe to aloha.png.
+* `find n\Jane doe` +
+Finds a saved contact named Jane Doe.
 
-==== Update Picture to Contact: `updatepic`
-You can update the profile picture of a contact in your address book.
+* `find n\Jane` +
+Finds all saved contacts named Jane.
```

Comment:
Finds all saved contacts with name containing "Jane"? <- Can be first name or last name.

Reply:
@xCelestialPhoenix Regarding the fullstop, I decided to make all notes to have no fullstop (since the notes at other features also does not have fullstop

Reply:
@xCelestialPhoenix yea it can be first name or last name, as long as it contains "Jane"